### ISSUE TITLE
Quasar ignores portconfiguration

### ISSUE THREAD

**Issue by MakeTeaNotLove** [scakemyer/plugin.video.quasar#744]
Quasar seems to ignore custom portconfiguration and listens on the default port of  6881 anyway. 

I've noticed Quasar asking the router for UPNP redirection on both the ports I've configured and the default port of 6881 as can be seen in the logs :

13:00:24  73.363976 T:1415574272  NOTICE: [plugin.video.quasar] NOTI  libtorrent   ▶ logAlerts        listen_succeeded_alert: successfully listening on [TCP] [::]:6881
13:00:24  73.410431 T:1415574272  NOTICE: [plugin.video.quasar] NOTI  libtorrent   ▶ logAlerts        listen_succeeded_alert: successfully listening on [TCP] 0.0.0.0:6881
13:00:24  73.430641 T:1415574272  NOTICE: [plugin.video.quasar] NOTI  libtorrent   ▶ logAlerts        listen_succeeded_alert: successfully listening on [UDP] 0.0.0.0:6881
13:00:24  73.451324 T:1415574272  NOTICE: [plugin.video.quasar] NOTI  libtorrent   ▶ logAlerts        listen_succeeded_alert: successfully listening on [TCP] [::]:14146
13:00:24  73.602417 T:1415574272  NOTICE: [plugin.video.quasar] NOTI  libtorrent   ▶ logAlerts        listen_succeeded_alert: successfully listening on [TCP] 0.0.0.0:14146
13:00:24  73.604912 T:1415574272  NOTICE: [plugin.video.quasar] NOTI  libtorrent   ▶ logAlerts        listen_succeeded_alert: successfully listening on [TCP] [::]:14125
13:00:24  73.611687 T:1415574272  NOTICE: [plugin.video.quasar] NOTI  libtorrent   ▶ logAlerts        listen_succeeded_alert: successfully listening on [TCP] 0.0.0.0:14125
13:00:24  73.613678 T:1415574272  NOTICE: [plugin.video.quasar] NOTI  libtorrent   ▶ logAlerts        listen_succeeded_alert: successfully listening on [UDP] 0.0.0.0:14125

1. Is this a bug in the software?
1.1 If not why is this type of behaviour desirable? I would argue that this type of behaviour strongly goes against the principle of least astonishment.

Env: Quasar 0.9.73, running on Kodi 2:16.1-5~jessie, running on Raspbian on a Raspberry Pi.

**Comment by scakemyer** [comment]
You really had to get fancy with your principle of least astonishment now did you... I mentioned libtorrent using port 6881 no matter what we set `listen_interfaces` to in #626. I'll look into it some more, but you should file an issue in libtorrent instead, and try to be less of smartass over there if you do. I agree that it'd be great to have this fixed nonetheless.

**Comment by MakeTeaNotLove** [comment]
Well that might be but I believe upnp-handling is quasar's responsibility or not? Sorry I get weird when I'm high :100:

**Comment by scakemyer** [comment]
Nope, we just set libtorrent settings, same thing with proxy settings, it's all managed by libtorrent.

**Comment by MakeTeaNotLove** [comment]
Ow, my bad. I'll have a look at libtorrent then. Thanks for all the work you've put into Quasar! I've found your bitcoin wallet and the .donate thingy ;)

**Comment by scakemyer** [comment]
Thanks! Might put a code bounty on this issue with it ;)
